org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorServletListener.mtl

```diff
@@ -88,7 +99,8 @@ public class [javaClassNameForAdaptorServletListener(anAdaptorInterface) /] impl
     public void contextInitialized(final ServletContextEvent servletContextEvent)
     {
         // [protected ('contextInitialized_init')]
-        //TODO: override the default servletName to match that defined in your web.xml configuration file.
+        //TODO: override any of the class constants as desired
```

Comment:
I think we should remove the TODO here and the sample code as this should now be the last resort given the new methods for setting the base declaratively.

Reply:
true.